src/Microsoft.AspNet.Razor/RazorResources.resx

```diff
@@ -425,4 +425,7 @@ Instead, wrap the contents of the block in "{{}}":
   <data name="TagHelperParseTreeRewriter_InvalidNestedTag" xml:space="preserve">
     <value>The &amp;lt;{0}&amp;gt; tag is not allowed by parent &amp;lt;{1}&amp;gt; tag helper. Only child tag helper(s) targeting tag name(s) '{2}' are allowed.</value>
   </data>
+  <data name="ParseError_HelperDirectiveNotAvailable" xml:space="preserve">
+    <value>The @helper directive is not available in Razor 4.</value>
```

Comment:
`The {0} directive is no longer supported.`

Don't want to specify the version for when we rev, and also don't want to inline `@helper` because it may be localized.